akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -932,6 +936,37 @@ directory. This can be changed by configuration where the specified path can be
 Note that it is not mandatory to specify a snapshot store plugin. If you don't use snapshots
 you don't have to configure it.
 
+.. _journal-proxy:
+
+Journal Proxy
+-------------
+
+A journal proxy allows sharing of journals and snapshot stores across multiple actor systems (on the same or on
+different nodes). This, for example, allows persistent actors to failover to a backup node and continue using the
+shared journal instance from the backup node. The journal proxy works by forwarding all the journal/snapshot store
+messages to a single, shared, persistence plugin instance, and therefor supports any use case supported by the proxied
+plugin (for example, :ref:`Persistence Query <persistence-query-scala>`).
```

Comment:
I've tested the LevelDB persistence query with the journal and it works (I also checked the code, and all queries forward messages to the journal).
I'm not sure that's actually what was intended for PQ, but I would argue that if it doesn't communicate with the persistence layer via the plugin then there's a pretty strong dependency on the exact data format the journal uses.